### ISSUE TITLE
numpy.bool_ not supported as a binding type

### ISSUE THREAD

**Issue by behanjc** [snowflakedb/snowflake-connector-python#153]
When trying to load data from a python tuple, I was receiving the error:
```
None: Failed processing pyformat-parameters; 255001: None: Binding data in type (bool_) is not supported
```

It looks like numpy.bool_ is not a supported binding type. Not sure if this is intentional for some reason.

**Comment by smtakeda** [comment]
not intentional. will fix it.

**Comment by smtakeda** [comment]
fixed in 1.8.0
